### DISCUSSION TITLE
remove .firebaserc from backend package

### DISCUSSION THREAD

**Discussion by ctrlc03** [quadratic-funding/mpc-phase2-suite#264] [General]

**Comment by 0xjei** [comment]
Why? This is more than useful to configure different projects. Have a look at #254

**Comment by ctrlc03** [comment]
should have been more specific, my bad. I would remove the default entry that is in the file, as after a fresh clone, a new user would face auth errors on firebase while setting up the project (functions deployment etc) as firebase would default to the project id included in this file.

**Comment by 0xjei** [comment]
Gotcha. Hmm IMHO the user will use the Firebase CLI for the deployment of a new project. The deploy script will use the name of the app indicated on the `.firebaserc` file as default. BTW, we are going to have two apps `prod` and `dev` with a different identifier. The user would be pretty much free to configure everything (must be specified in the deploy guide) and to select `dev` or `prod` projects when deploying.

**Reply by ctrlc03** [reply]
Right makes sense! let's make sure it is well documented
